input/vocabulary/codesystem-knowledgeArtifact-release-version-behavior.json

```diff
@@ -0,0 +1,30 @@
+{
+  "resourceType": "CodeSystem",
+  "id": "knowledgeArtifact-release-version-behavior",
+  "url": "http://hl7.org/fhir/uv/cmi/CodeSystem/knowledgeArtifact-release-version-behavior",
+  "version": "3.0.0",
+  "name": "KnowledgeArtifactReleaseVersionBehavior",
+  "title": "Knowledge Artifact Release Version Behavior Codes",
+  "status": "active",
+  "experimental": false,
+  "date": "2023-02-03T23:21:03-06:00",
+  "publisher": "Clinical Decision Support Workgroup",
+  "description": "Behaviors of 'version' parameter application in the $release operation.",
+  "caseSensitive": true,
+  "valueSet": "http://hl7.org/fhir/uv/cmi/ValueSet/knowledgeArtifact-release-version-behavior",
+  "content": "complete",
+  "concept": [
+    {
+      "code": "default",
+      "display": "Default"
```

Comment:
Codesystem should have definitions for each code that clearly define the intent of the code and it's meaning.